### ISSUE TITLE
Allow swapping of configurations

### ISSUE THREAD

**Issue by grosch** [vapor/apns#13]
Please provide a way that a single server app can support both the normal app's push notification (i.e. via the p8) and also passkit notifications (i.e. via the PEM files).

Right now we can't swap between configurations, to the best of my knowledge, because one thread might set it to config A, then another thread sets it to B, and then thread A tries to do the push and fails.

**Comment by tanner0101** [comment]
We should allow registering multiple configurations based on some identifier. For example:

```swift
app.apns.configuration[.foo] = ...

req.apns(.foo).send(..., to: ...)
```

**Comment by khanlou** [comment]
This would also be really great for being able to switch between production and sandbox targets based on the auth token

**Comment by 0xTim** [comment]
I don't believe this has progressed yet. Feel free to open a PR if you can though!

**Comment by kylebrowning** [comment]
This is fixed in #46
